content/guides/userguides/consumer-vignettes/export-to-terra.md

```diff
@@ -6,115 +6,146 @@ subTitle: ""
 
-For more information about how to use the Dockstore, see the [Dockstore documentation](https://docs.dockstore.org/en/develop/).
+## Tutorial — Automated processing of HCA BAM files in Terra 
 
-## Step one: finding BAM files with the HCA Data Explorer
+This tutorial guides you through finding and setting up a simple example workflow in Terra that you can use to process BAM files exported from the HCA Data Explorer.
 
-You can use the <link-to-browser relativelink="/projects">HCA Data Explorer</link-to-browser> to find data to export to Terra.
-The Data Explorer lists projects with data available for download from the Data Store and lets you filter the data for a number of attributes.
+For an additional Terra tutorial on using community tools with HCA data, see the [Intro-to-HCA-data-on-Terra workspace](https://app.terra.bio/#workspaces/featured-workspaces-hca/Intro-to-HCA-data-on-Terra). 
+
+### Step one: finding BAM files with the HCA Data Explorer and exporting to Terra
+
+You can use the <link-to-browser relativelink="/projects">HCA Data Explorer</link-to-browser> to find data to export to Terra. The Data Explorer lists projects with data available for download and lets you filter the data for a number of attributes.
 
 Using the Data Explorer, select some data that you are interested in. Choose anything that looks interesting - we will be running a really simple workflow that generates MD5 checksums of files, so the type of data is not important.
-When you have found a data set of interest, click on the big blue *Export Selected Data* button at the top right of the page. You will see something like this:
 
-<figure-styles width="710">
+When you have found a data set of interest, use the following steps:
+1. Click on the big blue *Export Selected Data* icon at the top right of the page. 
+1. Click on the *Analyze in Terra* icon at the bottom.
 
-![The *Export Selected Data* button](../../_images/terra-export_button.png)
+	![](../../_images/analyze-in-terra-export.png)
 
-</figure-styles>
+1. Click on the species you want to analyze.
+1. Select the files you want to export by clicking the checkbox next to the file type.
+1. Select the download format (the terra.pfb option is in development).
+1. Click the *Request Link* icon.
+	
+	You'll be redirected to a Data Portal page containing an export link.
 
-Click on the *Export to Terra* button. You will then see a page like this where you can select what kind of data to export:
+1. Click the workspace URL link.
+	
+	You'll be redirected to the Terra import page.
 
-<figure-styles shadowless=true>
+1. If you've already created a Terra workspace, select workspace by choosing an existing workspace or choose "Start with a new workspace."
 
-![Page for choosing data to export](../../_images/terra-choose_files.png)
 
-</figure-styles>
+	![](../../_images/terra-import.png) 
 
-Again, choose anything that looks interesting.
+### Step two: finding a workflow in Dockstore
```

Comment:
It might be worth adding a link to a WDL resource somewhere in this section.

Reply:
Good idea! Added.